scripts/dev_scripts/integration_tests.sh

```diff
@@ -99,6 +99,46 @@ $RUN_MACARON analyze -rp https://github.com/timyarkov/docker_test -b main -d 404
 
 python $COMPARE_JSON_OUT $JSON_RESULT $JSON_EXPECTED || log_fail
 
+echo -e "\n----------------------------------------------------------------------------------"
+echo "uiv-lib/uiv: Analysing the repo path, the branch name and the commit digest for an NPM project,"
```

Comment:
`npm` should be in lower case.
```suggestion
echo "uiv-lib/uiv: Analysing the repo path, the branch name and the commit digest for an npm project,"
```